openapi.yaml

```diff
@@ -471,6 +591,173 @@ paths:
           $ref: "#/components/responses/InternalServerError"
 components:
   schemas:
+    AddRelationArgs:
+      type: object
+      properties:
+        endpoints:
+          type: array
+          items:
+            type: string
+        via-cidrs:
+          type: array
+          items:
+            type: string
+      additionalProperties: false
+      required:
+        - endpoints
+    Application:
+      type: object
+      description: Information about a given application.
+      properties:
+        base:
+          $ref: '#/components/schemas/Base'
+        channel:
+          type: string
```

Comment:
I wonder if we should have a `CharmChannel` object that looks like:

```json
{
  "track": "latest",
  "channel": "edge",
  "branch": "pr-45"
}
```

WDYT?

Reply:
Makes sense, but with what you probably meant `"risk": "edge"` (instead of `"channel":  "edge"`). Also are the branches still in use?

Reply:
Yes, you're right that it should probably be:

```javascript
{
  "track": "latest",
  "risk": "edge",
  "branch": "pr-45"
}
```

And yes, branches are still in use! :)